src/main/kotlin/fi/metatavu/metaform/server/persistence/model/MetaformReplyViewed.kt

```diff
@@ -0,0 +1,29 @@
+package fi.metatavu.metaform.server.persistence.model
+
+import java.time.OffsetDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+class MetaformReplyViewed {
+
+    @Id
+    var id: UUID? = null
```

Comment:
This field does not actually exist, so use reply as Id

Reply:
Resolved